tsconfig.json

```diff
@@ -10,7 +10,8 @@
 			"es2015.promise",
 			"es2015.iterable",
 			"es2015.symbol",
-			"es2015.symbol.wellknown"
+			"es2015.symbol.wellknown",
+			"es2017"
```

Comment:
I'm not sure we should include `es2017`, this will allow writing code in the app that is not supported by browsers and error at runtime (not fun). I assume you included it for scripts, if that's the case we shouldn't share the tsconfigs.